### ISSUE TITLE
Source control warning message cannot be read

### ISSUE THREAD

**Issue by Thalos12** [mrworkman/papercolor-vscode-redux#1]
I like your theme, but I noticed that with the default colors the warning message issued in the source control tab when the commit line is too long cannot be read.
What I see is pictured below

![immagine](https://user-images.githubusercontent.com/13919407/76082381-898d3300-5fab-11ea-8413-a598dfaa7efc.png)

To fix it I added the following to my User settings 

```json
"workbench.colorCustomizations": {
        "[PaperColor Redux]": {
            "inputValidation.warningBackground": "#fffec6"
        },
},
```

and the message can now be read

![immagine](https://user-images.githubusercontent.com/13919407/76082783-69aa3f00-5fac-11ea-9f85-36241ecd2f6d.png)

Should either the background or the foreground be changed in `themes/papercolor-vscode-redux.json`?

**Comment by mrworkman** [comment]
It would probably make sense to change the background, as a black background is not really in keeping with the overall theme. 

I will try to publish an update soon. Thanks for attaching screenshots :)

**Comment by mrworkman** [comment]
Fixed. Released as v0.8.0.

Example:
![image](https://user-images.githubusercontent.com/20093062/76105209-6de65480-5fcc-11ea-9c6a-4b5e325cbc63.png)

**Comment by Thalos12** [comment]
Thank you for fixing it so fast!

Il ven 6 mar 2020, 18:02 Stephen Workman <notifications@github.com> ha
scritto:
